Kickstarter-iOS/Views/Controllers/PledgeViewController.swift

```diff
@@ -142,6 +155,17 @@ final class PledgeViewController: UIViewController,
     self.viewModel.inputs.viewDidLoad()
   }
 
+  override func viewDidLayoutSubviews() {
+    super.viewDidLayoutSubviews()
+
+    self.rootScrollView.contentInset = UIEdgeInsets(
```

Comment:
Adjusting content insets instead of pinning the scrollview to the top of the button container.